.gitignore

```
@@ -1,10 +1,28 @@
-/_build
-/cover
-/deps
-/doc
+# The directory Mix will write compiled artifacts to.
+/_build/
+
+# If you run "mix test --cover", coverage assets end up here.
+/cover/
+
+# The directory Mix downloads your dependencies sources to.
+/deps/
+
+# Where third-party dependencies like ExDoc output generated docs.
+/doc/
+
+# Ignore .fetch files in case you like to edit your project deps locally.
 /.fetch
+
+# If the VM crashes, it generates a dump, let's ignore it too.
 erl_crash.dump
+
+# Also ignore archive artifacts (built via "mix archive.build").
 *.ez
-*.beam
-/config/*.secret.exs
-.elixir_ls/
+
+# Ignore package tarball (built via "mix hex.build").
+papelillo-*.tar
+
```

Remove the .DS_Store file from the repo and ignore it ( this is a mac file and is not needed in the repo)

```suggestion
.DS_Store
```

---

I don't know why the difference is not shown in the comparison. If you see the file in the branch is there